### ISSUE TITLE
Issues: AttributeError in `create_anim.py`

### ISSUE THREAD

**Issue by SIKtt** [IhmeGroup/quail#40]
In post-process scripts `create_anim.py`
```
imgs = ax.collections.copy()
```
It comes with an Error
```
File "/quail-main/examples/scalar/2d/constant_advection/create_anim.py", line 35, in <module>
    imgs = ax.collections.copy()
AttributeError: 'ArtistList' object has no attribute 'copy'
```
this method seems changed in matplotlib 3.5.x [Release notes](https://matplotlib.org/stable/users/release_notes.html)
Maybe it can be fixed with:
```
imgs = ax.collections
```

**Comment by ericjching** [comment]
Thanks for opening the issue and suggesting a fix, @SIKtt!

**Comment by ericjching** [comment]
Fixed with #41
